frontend/pages/beta/settings.tsx

```diff
@@ -0,0 +1,62 @@
+import Box from '@mui/material/Box'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import React, { useState } from 'react'
+
+import { useGetCurrentUser } from '../../actions/user'
+import SettingsProfileTab from '../../src/settings/beta/SettingsProfileTab'
+import ThemeTab from '../../src/settings/beta/ThemeTab'
+import Wrapper from '../../src/Wrapper.beta'
+
+function TabPanel({ children, value, index, ...rest }) {
+  return (
+    <Box
+      role='tabpanel'
+      hidden={value !== index}
```

Comment:
I understand the statement, but what does it do for this attribute in his component?  If that makes sense

Reply:
`hidden` shouldn't be used. It's not a MUI or React attribute, but a default HTML attribute. It's best to avoid using `hidden` because it will be overridden if a `display` attribute is added to the element (e.g. `display: 'flex'` or `display: 'inline-block'`).

It's always better to use either:
1. Conditional rendering (preferred) - `{ value === index & <div>Hello World</div> }`
2. The display property - `<Box sx={{ display: value !== index ? 'none' : undefined }}>`

The following SO post explains in a bit more detail: https://stackoverflow.com/a/65685345